CHANGES.rst

```diff
@@ -41,6 +41,10 @@ New features
 
 Changes
 -------
+
+- The score(s) columns of :class:`ParamSearch.results_` is now the right-most columns.
+  :pr:`1378` by :user:`Vincent Maladiere <Vincent-Maladiere>`.
+
```

Comment:
```suggestion
```

the ParamSearch has never been released yet so this is not a change from users' point of view.

We will still want to mention all the recent prs related to expressions but it will be easier to create one summary changelog entry when preparing the release